website/static/js/search.js

```diff
@@ -78,6 +94,54 @@ var ViewModel = function(params) {
     self.searchCSS = ko.observable('active');
     self.onSearchPage = true;
 
+    self.licenses = ko.observableArray(
```

Comment:
Does this array ever change? If not, then it doesn't need to be an observableArray.